CONTRIBUTING.md

```diff
@@ -146,6 +146,12 @@ Three additional helpers exist that are refreshed every second:
 There is also a search bar in the top right of the dashboard. This fuzzy-searches
 image mocks based on their file name and trace type.
 
+#### Step 6b: Create a mock to test new features
+Create a new JSON inside
+`test\image\mocks\`
+which you'll then be able to search from the test dashboard
```

Comment:
In several cases adding new image tests are not necessary.
So this step should be optional and it could be part of `Step 6`.
You may revert changes in `CONTRIBUTING.md` and possibly submit a separate PR to suggest improvements as well.
Thank you!